app/controlplane/internal/server/grpc.go

```diff
@@ -254,7 +254,7 @@ func allowListEnabled() selector.MatchFunc {
 }
 
 func requireRobotAccountMatcher() selector.MatchFunc {
-	const requireMatcher = "controlplane.v1.AttestationService/.*|controlplane.v1.AttestationStateService/.*|controlplane.v1.SigningService/.*"
+	const requireMatcher = "controlplane.v1.AttestationService/.*|controlplane.v1.AttestationStateService/.*|controlplane.v1.SigningService/GenerateSigningCert"
```

Comment:
This is needed, since the new endpoint should be available for `workflow run describe`. This setting will make it public, but we might want to move it to its own service.

Reply:
I think it's fine since it can enable offline verification